### PR TITLE
kola/test/misc/raid.go: add coreos.disk.raid.data

### PR DESCRIPTION
**Pull request by dgonyeo**
Adds a test that raids together the OEM-CONFIG and USR-B partitions and mounts them at `/var/lib/data`. This test does not test if the raid array still functions after a reboot, therefore `/usr/share/oem/grub.cfg` doesn't need to be overwritten and it should be able to run on all platforms.

### CODE REVIEW

**Comment by dgonyeo** [comment]
After poking some more, I discovered that raid arrays are assembled at some point after root is mounted and before systemd attempts to run mount units, so this test can reboot and check that the array is still mounted just fine. I've updated this PR to do so.

File: kola/tests/misc/raid.go [hunk 1]
```diff
@@ -77,6 +110,22 @@ func RootOnRaid(c cluster.TestCluster) {
 	checkIfMountpointIsRaid(c, m, "/")
 }
 
+// FIXME(dgonyeo): once `rd.auto` is set by default this test should reboot and
```
Comment (bgilbert) [inline kola/tests/misc/raid.go]: stale

Comment (dgonyeo) [inline kola/tests/misc/raid.go]: fixed

**Comment by dgonyeo** [comment]
Good to merge, lack access to do so

**Pull request merged**